db/migrate/20220527143034_create_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users, id: :uuid do |t|
+      t.string :name, index: { unique: true, name: 'unique_name' }, null: false
+      t.string :email, index: { unique: true, name: 'unique_email' }, null: false
```

Comment:
We usually use automatically generated names for index names. But you can leave it as it is, since you are exploring rube/rails. You option is also 👍 .

```suggestion
      t.string :email, index: { unique: true }, null: false
```